### PR TITLE
a websocket notifying with a filter

### PR DESCRIPTION
**Pull request by jonenst**
Signed-off-by: Jon Harper <jon.harper87@gmail.com>

### CODE REVIEW

**Comment by geofjamg** [comment]
@jonenst could you add license and authorship in all your files?

File: src/test/java/org/gridsuite/notification/server/NotificationWebSocketIT.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+@DirtiesContext
+public class NotificationWebSocketIT {
+
+    @LocalServerPort
+    private String port;
+
+    @Test
+    public void echo() {
+        WebSocketClient client = new StandardWebSocketClient();
+        client.execute(getUrl("/notify"), WebSocketSession::close).block();
```
[the remark this answers is not in the document]
Comment (jonenst) [reply src/test/java/org/gridsuite/notification/server/NotificationWebSocketIT.java:34]: We just test that we can do the websocket handshake on the endpoint and close the connection without exceptions

File: src/main/java/org/gridsuite/notification/server/NotificationWebSocketConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import org.springframework.web.reactive.HandlerMapping;
+import org.springframework.web.reactive.handler.SimpleUrlHandlerMapping;
+import org.springframework.web.reactive.socket.WebSocketHandler;
+import org.springframework.web.reactive.socket.server.support.WebSocketHandlerAdapter;
+
+@Configuration
+public class NotificationWebSocketConfiguration {
+
+    @Autowired
+    @Qualifier("NotificationWebSocketHandler")
```
[the remark this answers is not in the document]
Comment (jonenst) [reply src/main/java/org/gridsuite/notification/server/NotificationWebSocketConfiguration.java]: removed qualifier, it's more in line with our regular coding style

**Comment by jonenst** [comment]
> @jonenst could you add license and authorship in all your files?

thanks done

**Pull request merged**